modules/deployments-replicationcontrollers.adoc

```diff
@@ -18,6 +18,11 @@ A selector is a set of labels assigned to the pods that are managed by the repli
 The replication controller does not perform auto-scaling based on load or traffic, as it does not track either. Rather, this requires its replica
 count to be adjusted by an external auto-scaler.
 
+[NOTE]
+====
+It is recommended to use a `DeploymentConfig` to create a replication controller. It is not recommended to create replication controllers directly.
```

Comment:
+1, and maybe we can add they should use replica sets if they want to create them directly

Reply:
@atiratree can you explain this a bit more? If you want to create <what> directly? (Because creating a replica set wouldn't create a replication controller I assume)

(And is it an okay idea to create replica sets directly, or is that not something we should actually recommend?)

Reply:
as we mention in the replica note something in this regard:

If you require custom update orchestration or do not require updates at all, use replica sets instead of replication controllers.